testing/generate_table1.py

```diff
@@ -0,0 +1,112 @@
+import pandas as pd
```

Comment:
A short description of the script could be added.

Reply:
done here https://github.com/ivadomed/model_seg_sci/pull/87/commits/7c8dc62a01a58b6b6cfb85efd2a576a76da55030!